2022/23xxx/CVE-2022-23632.json

```diff
@@ -1,18 +1,93 @@
 {
```

Comment:
'ttps://github.com/traefik/traefik/releases/tag/v2.6.1' does not match '^(ftp|http)s?://\\S+$'

Reply:
Fixed URL typo